src/main/java/com/minecolonies/coremod/permissions/ColonyPermissionEventHandler.java

```diff
@@ -318,15 +368,14 @@ public void on(final ArrowLooseEvent event)
      * Deny if:
      * - If the attacking happens in the colony
      * - Player is less than officer to the colony.
-     * - Entity is a citizen.
      *
      * @param event EntityItemPickupEvent
      */
     @SubscribeEvent
     public void on(final AttackEntityEvent event)
     {
         final EntityPlayer playerIn = event.getEntityPlayer();
-        if (Configurations.enableColonyProtection && colony.isCoordInColony(playerIn.getEntityWorld(), playerIn.getPosition()) && event.getTarget() instanceof EntityCitizen )
+        if (Configurations.enableColonyProtection && colony.isCoordInColony(playerIn.getEntityWorld(), playerIn.getPosition()))
```

Comment:
On the server we had a lot of cases of dead colonists, empty armor stands and dead livestock